src/client/containers/Profile/index.js

```diff
@@ -1,5 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
@pdsorensen Wouldn't be better to create a component in the component folder, then import it in the index.js.. then we don't add index.css here. :question:

Reply:
@arya56 that sounds reasonable.